Tests/Acceptance/ClearRevokedTokensCommandTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Tests\Acceptance;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use Trikoder\Bundle\OAuth2Bundle\Manager\AccessTokenManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\AuthorizationCodeManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\ClientManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\RefreshTokenManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\ScopeManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Model\AccessToken;
+use Trikoder\Bundle\OAuth2Bundle\Model\AuthorizationCode;
+use Trikoder\Bundle\OAuth2Bundle\Model\RefreshToken;
+use Trikoder\Bundle\OAuth2Bundle\Tests\Fixtures\FixtureFactory;
+
+final class ClearRevokedTokensCommandTest extends AbstractAcceptanceTest
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        timecop_freeze(new \DateTimeImmutable());
```

Comment:
@franjo-zadelj-trikoder why is the time freezed for this test?

Reply:
@X-Coder264 why is it frozen here vendor/trikoder/oauth2-bundle/Tests/Acceptance/ClearExpiredTokensCommandTest.php

Reply:
@franjo-zadelj-trikoder Because whether the token is expired or not depends on the time, which is not the case with the revoke functionality.